### DISCUSSION TITLE
Picking PDF viewer for preview

### DISCUSSION THREAD

**Discussion by lokesh-krishna** [quarto-dev/quarto-nvim#134] [Q&A]
Currently previewing PDFs defaults to Firefox on my system. I'd like it to instead use Zathura. Is there a way to pick which program is used to preview PDFs?

**Comment by jmbuhr** [comment]
This question is better suited for the quarto-cli repo, but from what I gathered, quarto always opens formats (that it can) with the default web browser (https://github.com/quarto-dev/quarto-cli/blob/8c3244c3ad4d850a14d38d5c9fc38ab12e90e1b2/src/command/preview/preview.ts#L259-L265). This it is doing because the server can then send a command to the browser to automatically reload when the file changes.

However, Zathura has its own filewatch capabilities. So you could use `quarto preview --no-browser` and then have Zathura watch the resulting pdf.
